eng/tools/generator/template/rpName/packageName/build.go.tpl

```diff
@@ -2,6 +2,6 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 // This file enables 'go generate' to regenerate this specific SDK
-//go:generate pwsh.exe ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate resourcemanager/{{rpName}}/{{packageName}}
+//go:generate pwsh.exe ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate -filePrefix "" resourcemanager/{{rpName}}/{{packageName}}
```

Comment:
Can we have default value for -filePrefix as empty instead of zz_generated_,  then we don't need to change the below files.

Reply:
For previous build.ps1, the default is `zz_generated_`. I don't quite sure it is safe to change it default to empty. So, I chose to change our build template.

Reply:
But as 'empty' is the default behaviour, so shall we update our script to make it default?
What do you think of the risk of this change?

Reply:
I've searched the code base and it seems only mgmt. plane uses build.ps1 to do the code generation. So, maybe it only has impact for our mgmt. SDK, I've changed the default behavior and will track with any possible issue related to it after merge.